dash-pipeline/SAI/Makefile

```diff
@@ -1,19 +1,6 @@
-dashone: test.cpp
-	g++ \
-	    -I SAI/inc \
-	    -I SAI/experimental/ \
-	    -o test test.cpp saidash.cpp \
-	    -L/usr/local/lib/ \
-	    -lpthread \
-	    -lpiprotogrpc \
-	    -lpiprotobuf \
-	    -lprotobuf \
-	    -lgrpc++ \
-	    -lgrpc \
-	    -lpiall \
-	    -lpi_dummy \
-	    -lpthread \
-	    -labsl_synchronization \
-	    -g
-
 
+all:
```

Comment:
I like this change, it's long overdue, The old make content was terribly obsolete, leftover from early days. Thanks!